### DISCUSSION TITLE
[2761570]红楼梦 Stone Story {本体7.19更新}

### DISCUSSION THREAD

**Discussion by zhana1028** [pjy612/SteamManifestCache#753] [Share]
### Steam 游戏ID

2761570

### 密钥列表

```hash
2761571;2ba0fc9f31d6320e7a7dc80ea7c85fdc92fa3f8d2bec09a2fb9b1a3ddc56086b
```

### 清单附件

[2761571_3210469073699894265.zip](https://github.com/user-attachments/files/16465265/2761571_3210469073699894265.zip)

### 备注

7月19日更新的最新版
email:lzh889@foxmail.com

**Comment by pjy612** [comment]
这是985 的那个游戏么。。。

**Reply by zhana1028** [reply]
我不知道啊，这个游戏有什么说法吗？

**Reply by zhana1028** [reply]
之前我分享的发完之后，看到重复了就删了，不知道这俩是不是也重复了。。

**Reply by pjy612** [reply]
这个没重复。。。别是自己买的 囧。。。
